src/style.css

```diff
@@ -0,0 +1,152 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card-apple--width {
+  width: 200px;
+}
```

Comment:
Create page block, and make card the element of this block
Then you can set width: 200px to the page__card element